### PR TITLE
LTS Version Change

### PR DESCRIPTION
**Pull request by Skptak**
Updating version number and CHANGELOG for LTS 2.0 Release

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Change Log for corePKCS11 Library
 
+## v3.5.0 (August 2022)
+* [#156](https://github.com/FreeRTOS/corePKCS11/pull/156) MISRA compliance updates
```
Comment (ravibhagavandas) [inline CHANGELOG.md:4]:
```suggestion
* [#156](https://github.com/FreeRTOS/corePKCS11/pull/156) MISRA C:2012 compliance updates
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
...
+* [#155](https://github.com/FreeRTOS/corePKCS11/pull/155) Update CBMC Starter kit
+* [#150](https://github.com/FreeRTOS/corePKCS11/pull/150) Add mbedtls 3.X support to core_pkcs11_mbedtls.c
```
Comment (ravibhagavandas) [inline CHANGELOG.md:6]: Is it backwards compatible with older versions ? If so please mention that as well.

File: CHANGELOG.md (hunk 1 above)
Comment (Skptak) [reply CHANGELOG.md:4]: Added that to the line, also did that in other repos as well.

File: CHANGELOG.md (hunk 2 above)
Comment (Skptak) [reply CHANGELOG.md:6]: It should be according to the PR, added it to the message.

**Review by ravibhagavandas — APPROVED** [review]

**Review by AniruddhaKanhere — APPROVED** [review]

**Pull request merged**